DNN Platform/Library/Common/Utilities/PathUtils.cs

```diff
@@ -205,10 +205,10 @@ internal string GetUserFolderPathInternal(UserInfo user)
         public virtual bool IsDefaultProtectedPath(string folderPath)
         {
             return String.IsNullOrEmpty(folderPath) ||
-                   folderPath.ToLowerInvariant() == "skins" ||
-                   folderPath.ToLowerInvariant() == "containers" ||
-                   folderPath.ToLowerInvariant().StartsWith("skins/") ||
-                   folderPath.ToLowerInvariant().StartsWith("containers/");
+                   folderPath.Equals("skins", StringComparison.InvariantCultureIgnoreCase) ||
+                   folderPath.Equals("containers", StringComparison.InvariantCultureIgnoreCase) ||
+                   folderPath.StartsWith("skins/", StringComparison.InvariantCultureIgnoreCase) ||
+                   folderPath.StartsWith("containers/", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
This just seems odd. Wouldn't it make more sense to switch to just two checks for contains instead of doing an equals or startswith?